packages/babel-helpers/src/index.js

```diff
@@ -257,12 +257,15 @@ function loadHelper(name) {
 
 export function get(
   name,
-  getDependency?: string => t.Expression,
+  getDependency?: string => ?t.Expression,
   id?,
-  getLocalBindings?: () => string[],
+  localBindings?: string[],
 ) {
-  const helper = loadHelper(name);
-  return helper(getDependency, id, getLocalBindings);
+  return loadHelper(name).build(getDependency, id, localBindings);
+}
+
+export function getDependencies(name: string): IterableIterator<string> {
```

Comment:
How do you feel about exposing iterables as part of a public API? To me, I'd prefer to stick with an Array to make life easier for consumers still, so I'd vote to wrap this with `Array.from()`

Reply:
I was in doubt because just returing the iterable is faster than doing `iterable -> array -> iterable`, but probably it can't be noticed since it is only run once per helper (premature optimizations 🚀 🤣)